tools/spy-detector/src/main.rs

```diff
@@ -0,0 +1,262 @@
+use dashmap::DashMap;
+use nng::options::protocol::pubsub::Subscribe;
+use nng::options::Options;
+use nng::{Protocol, Socket};
+use shared::clap;
+use shared::clap::Parser;
+use shared::event_msg;
+use shared::event_msg::event_msg::Event;
+use shared::net_msg;
+use shared::prost::Message;
+use shared::simple_logger;
+use std::sync::atomic;
+use std::sync::atomic::AtomicU32;
+use std::sync::Arc;
+use std::time::Duration;
+use std::time::Instant;
+
+const ADDRESS: &str = "tcp://127.0.0.1:8883";
```

Comment:
This should be a clap argument. See e.g.

https://github.com/0xB10C/peer-observer/blob/8a445f71608bfd968c1231fbfa5b5864f06ab53c/tools/websocket/src/main.rs#L22-L24

Reply:
Okay